### PR TITLE
Increase targetBufferSize so that 99% of ReadB64 calls fit into the b…

### PR DESCRIPTION
**Pull request by patrickxb**
…uffer

### CODE REVIEW

**Comment by patrickxb** [comment]
cc @chrisnojima

**Comment by patrickxb** [comment]
i'm trying this now on 4.3.0 (22) ... can't tell a difference.

**Comment by chrisnojima** [comment]
We can merge and revert if it's weird

File: go/bind/keybase.go [hunk 1]
```diff
@@ -342,6 +342,10 @@ func ReadB64() (res string, err error) {
 	}
 	n, err := conn.Read(buffer)
 	if n > 0 && err == nil {
+		if n == bufferSize {
+			kbCtx.Log.Info("ReadB64 %d bytes (full buffer)", n)
```
Comment (mmaxim) [inline go/bind/keybase.go]: Is this output useful?

Comment (patrickxb) [reply go/bind/keybase.go]: I removed it.

**Pull request merged**